.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@ repos:
   rev: v2.2.0
   hooks:
     - id: commitlint
-      stages: [commit-msg]
+      stages: [merge-commit, commit-msg]
```

Comment:
Does the order matter here and we explicitly specify this order for a reason?

Reply:
My fault, we don't need it. It's involved during local test